partcad/src/partcad/part_factory_brep.py

```
@@ -0,0 +1,134 @@
+            raise
+
+    async def _process_brep_subprocess(self):
+        """
+        Process a BREP file using a subprocess for larger files.
+        """
+        # Initialize runtime if not already done
+        if self.runtime is None:
+            pc_logging.debug("Initializing subprocess runtime...")
+            self.runtime = self.ctx.get_python_runtime(self.PYTHON_RUNTIME_VERSION)
+            if self.runtime is None:
+                raise RuntimeError("Failed to initialize runtime for subprocess execution.")
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)